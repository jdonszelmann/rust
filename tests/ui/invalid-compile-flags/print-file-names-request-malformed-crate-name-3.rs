//@check-pass
//@ compile-flags: --print=file-names

#[crate_name = concat!("wrapped")]

macro_rules! inline {
    () => {};
}
