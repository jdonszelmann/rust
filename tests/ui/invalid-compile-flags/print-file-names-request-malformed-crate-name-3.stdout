print-file-names-request-malformed-crate-name-3
